### PR TITLE
Allow publishing drafts via the interface

### PR DESCRIPTION
**Pull request by ashmaroli**
Publish a draft with a couple of clicks..

### CODE REVIEW

File: src/containers/views/DraftEdit.js [hunk 1]
```diff
@@ -157,6 +195,13 @@ export class DraftEdit extends Component {
                 block
               />
               <Splitter />
+              <Button
+                onClick={() => this.handleClickPublish(path)}
+                type="publish"
+                icon="send-o"
+                active={true}
```
Comment (mertkahyaoglu) [inline src/containers/views/DraftEdit.js]: No need for `true`. Just pass `active`.

File: src/ducks/drafts.js [hunk 2]
```diff
@@ -106,6 +111,24 @@ export const deleteDraft = (directory, filename) => dispatch => {
     );
 };
 
+export const publishDraft = (directory, filename) => (dispatch, getState) => {
+  const metadata = getState().metadata.metadata;
+  let { raw_content, title } = metadata;
```
Comment (mertkahyaoglu) [inline src/ducks/drafts.js]: This isn't changing. We can make it `const`?

Also, `title` is not used?

File: src/containers/views/DraftEdit.js (hunk 1 above)
Comment (ashmaroli) [reply src/containers/views/DraftEdit.js]: This was in sync with existing JSX on `master`:
https://github.com/jekyll/jekyll-admin/blob/a592eafc18611eae46a0182a9cae9f203854b69e/src/containers/views/DraftEdit.js#L152-L166

I'll send a PR to fix `master` first..

File: src/ducks/drafts.js (hunk 2 above)
Comment (ashmaroli) [reply src/ducks/drafts.js]: Oops.. Good catch. :+1:

**Comment by ashmaroli** [comment]
As per current implementation, a draft's title won't be used to determine the generated post's filename.
Only the draft's basename (with extension) is considered.

We *may* use the draft's title to determine generated post's title in a future release.

**Pull request merged**